### PR TITLE
fix(bin): ensure progressbar works for stdout

### PR DESCRIPTION
**Pull request by dignifiedquire**
also redirects logs to STDERR

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -266,6 +266,19 @@ async fn get_interactive(
         .await;
 
     let pb = ProgressBar::hidden();
+    pb.enable_steady_tick(std::time::Duration::from_millis(1));
```
Comment (rklaehn) [inline src/main.rs]: Isn't 1 millisecond a bit much? I would think that 50ms or even 500ms would still be fine.

**Review by rklaehn — APPROVED** [review]

File: src/main.rs (hunk 1 above)
Comment (dignifiedquire) [reply src/main.rs]: you are right, set to 50ms for now 🤷

File: src/main.rs [hunk 2]
```diff
@@ -333,18 +342,18 @@ async fn get_interactive(
                 })
                 .await??;
                 let file = tokio::fs::File::from_std(dup);
-                let out = tokio::io::BufWriter::new(file);
-                // wrap for progress bar
-                let mut wrapped_out = pb.wrap_async_write(out);
-                tokio::io::copy(&mut reader, &mut wrapped_out).await?;
+                let mut file_buf = tokio::io::BufWriter::new(file);
```
Comment (flub) [inline src/main.rs:347]: The default buffer size is only 8KiB.  I think on most modern systems you can get a lot faster when writing to disk with something larger.  IIRC gnu coreutils defaults to 128KiB which they deem a good compromise for most cases, so I think that's a more reasonable default.

**Review by flub — APPROVED** [review]

File: src/main.rs (hunk 2 above)
Comment (dignifiedquire) [reply src/main.rs:347]: We have a bunch of these lying around, so lets change that/test that independently

**Pull request merged**